.npmrc

```diff
@@ -1 +1,2 @@
-puppeteer_skip_chromium_download = true
+puppeteer_skip_chromium_download=true
+legacy-peer-deps=true
```

Comment:
If no in this PR, we should still update peer-depts one day

Reply:
Sure - I want to keep changes to a minimum and then introduce proper deps in next PR

Reply:
Can you create a task for this so it's doesn't get forgotten?

Reply:
https://github.com/saleor/saleor-dashboard/issues/2549